src/pages/match/MatchSighting.jsx

```diff
@@ -0,0 +1,206 @@
+  });
+
+  const queryAnnotations = useMemo(
+    () => {
+      const annotationData = get(matchResults, 'annotation_data', {});
+      const originalQueryAnnotations = get(
+        matchResults,
+        'query_annotations',
+        [],
+      );
+      return originalQueryAnnotations.map(annotSageData => {
+        const annotHoustonData = get(
+          annotationData,
+          annotSageData?.guid,
+          {},
+        );
+        return {
+          ...annotHoustonData,
+          ...annotSageData,
+        };
+      });
+    },
+    [matchResults],
+  );
+
+  const matchCandidates = useMemo(
+    () => {
+      const hotspotterAnnotationScores = get(
+        selectedQueryAnnotation,
+        ['algorithms', 'hotspotter_nosv', 'scores_by_annotation'],
+        [],
+      );
+      return hotspotterAnnotationScores.map(scoreObject => {
+        const matchingAnnotation = get(matchResults, [
+          'annotation_data',
+          scoreObject?.guid,
+        ]);
+        return {
+          ...matchingAnnotation,
+          ...scoreObject,
+        };
+      });
+    },
+    [matchResults, selectedQueryAnnotation],
+  );
+
+  const loading = sightingDataLoading || matchResultsLoading;
+  const error = sightingDataError || matchResultsError;
+
+  if (loading) return <LoadingScreen />;
+  if (error) return <SadScreen variant="genericError" />;
+
+  const idCompleteTime = sightingData?.unreviewed_start_time;
+  const formattedIdTime = formatDate(
+    idCompleteTime,
+    true,
+    'Unknown time',
+  );
+
+  const matchPossible =
+    selectedMatchCandidate && selectedQueryAnnotation;
+
+  return (
+    <MainColumn
+      fullWidth
+      style={{ maxWidth: 'unset', padding: '0 16px' }}
+    >
+      <div
+        style={{
+          padding: '16px 0',
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <div>
+          <ButtonLink
+            href={`/sightings/${sightingGuid}`}
+            display="back"
+            id="BACK_TO_SIGHTING"
+          />
+          <Text variant="h4">Review match candidates</Text>
+          <Text
+            variant="subtitle2"
+            id="IDENTIFICATION_FINISHED_TIME"
+            values={{ time: formattedIdTime }}
+            style={{ padding: '2px 0 0 2px' }}
+          />
+        </div>
+        <ButtonMenu
+          buttonId="match-actions"
+          style={{ marginTop: 44 }}
+          actions={buttonActions}
+        >
+          Actions
+        </ButtonMenu>
+      </div>
+      <div style={{ display: 'flex' }}>
+        <div
+          style={{
+            width: '50%',
+            paddingRight: 0.5 * spaceBetweenColumns,
+          }}
+        >
+          <ImageCard
+            titleId="SELECTED_QUERY_ANNOTATION"
+            annotation={selectedQueryAnnotation}
+          />
+          <QueryAnnotationsTable
+            queryAnnotations={queryAnnotations}
+            selectedQueryAnnotation={selectedQueryAnnotation}
```

Comment:
This prop isn't used in the table. Will it be in the future, or should this be removed?

Reply:
Will be used in the future when the table is a properly controlled component